resources/js/Components/AuthWidget.vue

```diff
@@ -1,32 +1,30 @@
 <template>
     <div class="auth-widget" v-if="user">
-        <WikitLink :href="`https://www.wikidata.org/wiki/User:${user.name}`">
+        <a class="cdx-docs-link" :href="`https://www.wikidata.org/wiki/User:${user.name}`">
             <img src="images/user.svg" class="icon-user" /><span class="username">{{ user.name }}</span>
-        </WikitLink>
-        <WikitLink href="/auth/logout">{{ $i18n('log-out') }}</WikitLink>
+        </a>
+        <a class="cdx-docs-link"  href="/auth/logout">{{ $i18n('log-out') }}</a>
     </div>
-    <WikitLink v-else href="/auth/login">{{ $i18n('log-in') }}</WikitLink>
+    <a class="cdx-docs-link" v-else href="/auth/login">{{ $i18n('log-in') }}</a>
 </template>
 
 <script lang="ts">
 
 import type { PropType } from 'vue';
 import { defineComponent } from 'vue';
 import User from '../types/User';
-import { Link as WikitLink } from '@wmde/wikit-vue-components';
 
 export default defineComponent({
     props: {
         user: Object as PropType<User>
-    },
-    components: {
-        WikitLink
     }
 });
 </script>
 
 <style lang="scss">
 @import '~@wmde/wikit-tokens/dist/_variables.scss';
+@import '~@wikimedia/codex-design-tokens/theme-wikimedia-ui.scss';
```

Comment:
Why are these imported here? Aren't they only needed in `link.scss`?

Reply:
not according to the codex docs -
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/7420952/68f9a438-aad4-4877-a086-5780fd131383) but i did try it out locally in one spot it doesn't seem to break so we might consider doing it anyway